code/controllers/subsystem/mapping.dm

```diff
@@ -119,6 +119,7 @@ var/datum/controller/subsystem/mapping/SSmapping
 	INIT_ANNOUNCE("Loading [config.map_name]...")
 	TryLoadZ(config.GetFullMapPath(), FailedZs, ZLEVEL_STATION)
 	INIT_ANNOUNCE("Loaded station in [(REALTIMEOFDAY - start_time)/10]s!")
+	feedback_add_details("station_name", config.map_name)
```

Comment:
should probably name the stat `map_name`, station name can be confused with the actual station name, imo